nbgrader/docs/source/configuration/student_version.rst

```diff
@@ -102,3 +107,69 @@ can be configured through the ``ClearSolutions.text_stub`` option:
 
     c.ClearSolutions.text_stub = "Please replace this text with your response."
 
+
+"Autograder tests" cells
+-------------------------
+
+.. versionadded:: 0.5.0
+
+Default behavior
+^^^^^^^^^^^^^^^^
+
+By default, ``nbgrader assign`` will remove regions beginning with
+``### BEGIN HIDDEN TESTS`` and ``### END HIDDEN TESTS``, for example:
+
+.. code:: python
+
+    def validate_answer():
+        assert squares(1) = [1]
+        ### BEGIN HIDDEN TESTS
+        assert squares(2) = [1, 4]
+        ### END HIDDEN TESTS
+
+    validate_answer()
+
+will be released as:
+
+.. code:: python
+
+    def validate_answer():
+        assert squares(1) = [1]
+
+    validate_answer()
+
+.. note::
+
+    Keep in mind that wrapping all tests (for an "Autograder tests" cell) in
+    this special syntax will remove all these tests in the release version and
+    the students will only see a blank cell. It is recommended to have at least
+    one or more visible tests, or a comment in the cell for the students to
+    see.
+
+Changing the defaults
+^^^^^^^^^^^^^^^^^^^^^
+
+If you need to change these defaults (e.g., if your class doesn't use Python,
+or isn't taught in English), the values can be configured in the
+:doc:`nbgrader_config.py <config_options>` file. Most relevant are the options
+to the ``ClearHiddenTests`` preprocessor, which is the part of nbgrader that
+actually clears the solutions when producing the student version of the
+notebook.
+
+You can specify solution delimeters for any languages you want by setting the
+``ClearHiddenTests.begin_test_delimeter`` and
+``ClearHiddenTests.end_test_delimeter`` config options, thus allowing you to
+include notebooks of different languages within the same assignment:
+
+.. code:: python
+
+    c = get_config()
+    c.ClearHiddenTests.begin_test_delimeter = {
+        "python": "### BEGIN HIDDEN TESTS",
+        "javascript": "// BEGIN HIDDEN TESTS"
+    }
+    c.ClearHiddenTests.end_test_delimeter = {
```

Comment:
This is wrong, `text_delimeter` configs are Unicode not Dict, need to update docs.
I think this may also be wrong in the docs for the clearsolutions preprocessor.